cli/proc_state.rs

```diff
@@ -191,11 +194,25 @@ impl ProcState {
     cli_options: Arc<CliOptions>,
     maybe_sender: Option<tokio::sync::mpsc::UnboundedSender<Vec<PathBuf>>>,
   ) -> Result<Self, AnyError> {
+    let dir = cli_options.resolve_deno_dir()?;
+    let caches = Caches::default();
+    // Warm up the caches we know we'll likely need based on the CLI mode
+    match cli_options.sub_command() {
+      DenoSubcommand::Run(_) => {
+        _ = caches.dep_analysis_db(&dir);
```

Comment:
I didn't know this syntax was allowed 😄

Reply:
I just learned about it recently (it's fairly new and recommended over `let _` I assume because it saves 4 bytes. 😄